services/src/services/errorTelemetry/errorTelemetry.class.js

```diff
@@ -27,9 +27,9 @@ class ErrorTelemetryService {
       return Promise.reject({status:'KO'});
     }
 
-    // FIXME: TBD where, what and how to actually send data
     const toWrite = `${new Date()} => ${JSON.stringify(data)}\n\n`;
     fs.appendFileSync(this.loggingFile, toWrite);
```

Comment:
The file appending can certainly be removed with the Sentry integration

What is going to have to change, and I'm curious what solution you and @batmat come up with, is how we would verify from our smoke test whether the Sentry logging is working correctly. While I don't think that we should be sending exceptions to a live Sentry instance, some level of validation that something _should_ have been passed to Sentry from a running smoke test is important IMO.

Reply:
I've been looking into a mocking option that will allow us to unit test the sentry class, I'll create a ticket for that to make sure it gets done.

Reply:
Also, I wouldn't remove the file appending until we're 100% on a final end solution.  Even then, it's useful to have a backup of what data has been received since we don't track it any other way.

Reply:
The file appending is not going to work in a production environment since we'll be deployed as a stateless set of containers in Kubernetes. We won't be able to sensibly retrieve the contents of the file system, but if we miss an errant exception or two, I won't be terribly sad.